subtests/docker_cli/run/run_install.py

```diff
@@ -20,10 +20,13 @@ def postprocess(self):
         # ancestor method must have been successful
         self.sub_stuff['containers'].append(name)
 
-        mustpass(DockerCmd(self, 'commit', [name, name]).execute())
+        # images with same names as containers are confusing
+        eman = name[-1::-1]
```

Comment:
Cute, but causes suffering to the maintenance programmer.

Also, please use `tolower()` to make sure there are no caps in the name. Caps lead to a docker barf on commit.